superset/models/core.py

```diff
@@ -411,7 +413,7 @@ class Dashboard(Model, AuditMixinNullable, ImportMixin):
     owners = relationship(security_manager.user_model, secondary=dashboard_user)
 
     export_fields = ('dashboard_title', 'position_json', 'json_metadata',
-                     'description', 'css', 'slug')
+                     'description', 'css', 'slug', 'uuid')
```

Comment:
Since we should have logic that ties to `uuid`, we could have the export function force the `uuid` to be exported and this way the user wouldn't have to think about it.

Reply:
Did this, SQLAJsonEncoder now looks for a 'uuid' field and adds it if present.

```
        if hasattr(o, '__dict__'):
            vals = {}
            if 'uuid' in o.__dict__.keys():
                vals['uuid'] = o.__dict__['uuid']
```